pkg/plugins/trigger/pull-request.go

```diff
@@ -219,5 +220,8 @@ func buildAll(c Client, pr *scm.PullRequest, eventGUID string, elideSkippedConte
 	if err != nil {
 		return err
 	}
+	if eventGUID == "" && os.Getenv("GIT_KIND") == "github" {
```

Comment:
Not sure how other scm providers deal with GUID, so let's keep it only for github